mlir/test/Dialect/Rock/lowering_blockwise_broadcast_reduce.mlir

```diff
@@ -0,0 +1,122 @@
+// RUN: rocmlir-opt -rock-blockwise-gemm-to-threadwise -canonicalize -split-input-file %s | FileCheck %s
+
+// CHECK: #[[MAP:.*]] = affine_map<(d0, d1) -> (d1, d0)>
+// CHECK: #[[MAP1:.*]] = affine_map<(d0, d1) -> (d1 * 20 + d0)>
+// CHECK: #[[MAP2:.*]] = affine_map<(d0, d1, d2) -> (d0 + d1, d2)>
+// CHECK: #[[MAP3:.*]] = affine_map<(d0, d1) -> (d0, d1)>
+// CHECK: #[[MAP4:.*]] = affine_map<(d0, d1) -> (0, d0)>
+// CHECK: #[[TMAP:.*]] = #rock.transform_map<#[[MAP]] {{.*}}
+// CHECK: #[[TMAP1:.*]] = #rock.transform_map<#[[MAP]] by {{.*}}
+// CHECK: #[[TMAP2:.*]] = #rock.transform_map<#[[MAP1]] by {{.*}}
+// CHECK: #[[TMAP3:.*]] = #rock.transform_map<#[[MAP2]] by {{.*}}
+// CHECK: #[[TMAP4:.*]] = #rock.transform_map<#[[MAP3]] by {{.*}}
+// CHECK: #[[TMAP5:.*]] = #rock.transform_map<#[[MAP3]] by {{.*}}
+// CHECK: #[[TMAP6:.*]] = #rock.transform_map<#[[MAP4]] by {{.*}}
+// CHECK: func @rock_blockwise_reducesum_nr_threads_gt_blocksize
```

Comment:
Nit: CHECK-LABEL

Reply:
We cant use that because CHECK-LABEL splits the scoping and we need stuff from above func for stuff below right ?

Reply:
If you go poking around the FileCheck docs there's ways to have global variables. I don't recall if there's a flag or if it's just leading `$`. I know I've used it in other tests.